src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -35329,6 +35407,8 @@ static class E
         var symbol = model.GetDeclaredSymbol(extensionParameter);
         Assert.Equal(SymbolKind.Parameter, symbol.Kind);
         Assert.Equal("System.Int32 i", symbol.ToTestDisplayString());
+
+        Assert.Equal("E", model.GetEnclosingSymbol(extensionParameter.SpanStart).ToTestDisplayString());
```

Comment:
>Assert.Equal("E", model.GetEnclosingSymbol(extensionParameter.SpanStart).ToTestDisplayString()); [](http://example.com/codeflow?start=8&length=96)

Is this consistent with method parameters? Do we get type when we call ```GetEnclosingSymbol``` in parameter list?